packages/x-date-pickers-pro/src/internals/hooks/useDesktopRangePicker/useDesktopRangePicker.tsx

```diff
@@ -26,6 +23,7 @@ import {
 } from '../useEnrichedRangePickerFieldProps';
 import { getReleaseInfo } from '../../utils/releaseInfo';
 import { useRangePosition } from '../useRangePosition';
+import { PickerRangePositionContext } from '../../../hooks/usePickerRangePositionContext';
```

Comment:
Isn't this import a level too deep?

Reply:
For relative import, I tend to use deep imports except for the models, because it reduces the risk of circular imports
Do you see a reason not to?

Reply:
No... I just thought we have a rule that prevents 3-level relative imports

Reply:
On absolute imports yes, because it does not work with most bundling system